### ISSUE TITLE
on the fly patch broken

### ISSUE THREAD

**Issue by powerpuffboysz** [libertyernie/vba-wii#12]
the original tantric version of vba gx had a feature where roms did not need to be patched with a utility. the ips/ups only had to be in the same folder and have the same name as the rom and the patch would apply on starting the game. your updates seem to have broken that as I can't launch mother 3 without either hard patching or removing my ups patch file from my rom folder or any other patches for that matter.

**Comment by libertyernie** [comment]
Are just UPS patches broken? I was able to get an IPS to work but haven't tried UPS yet.

**Comment by powerpuffboysz** [comment]
I actually tried again and found less crashing.
http://api.viglink.com/api/click?format=go&jsonp=vglnk_14266361952468&drKey=1082&libId=i7dycie501000bl0000DA19m2jhg9kedp6&loc=http%3A%2F%2Fforums.therockmanexezone.com%2Ftopic%2F8886349%2F1%2F&v=1&out=https%3A%2F%2Fdl.dropbox.com%2Fs%2Fzrsgd1uk822jzf0%2FTranslation%2520Patch.zip&title=RockMan%20EXE6%20Falzar%20Translation%20Patch&txt=https%3A%2F%2Fdl.dropbox.com%2Fs%2Fzrsgd1uk822jzf0%2FTranslation%2520Patch.zip for megaman battle network 6 falzar (j) and mother 3 don't work period though so I'm still going to assume that the feature is broken. at least on larger patches

**Comment by libertyernie** [comment]
Yeah, I could tell the IPS patch worked by the title screen. So it must just be UPS that's broken.
I noticed that this patch in VBA-M was not included in VBA-GX originally, but is now since I updated it: http://sourceforge.net/p/vbam/code/901/tree//trunk/src/common/Patch.cpp?diff=835

Try this modified version of 2.3.2 with the old UPS patch code, and let me know how it works: https://www.dropbox.com/s/imbvua6e8u62529/VBA%20GX%202.3.2%20with%20old%20UPS%20patch%20behavior.dol?dl=0

**Comment by powerpuffboysz** [comment]
nope still had same results with smaller ips patches working and with larger ips and ups patches still crashing. did you try the patch from my last response for mmbn6?

**Comment by libertyernie** [comment]
I haven't tried that patch yet. But if that didn't help, or if IPS 
patches don't work sometimes, I really don't know what the problem could 
be - I'm not aware of any other changes in the patching code that I 
could try making. (Most of the code is a mystery to me - I just got it 
from VBA-M and made it replace the old VBA-M code from when the original 
developer put it in.)

**Comment by powerpuffboysz** [comment]
ok sorry for bothering if you can't fix it
